classes/class4/homework-soln/public/javascripts/handlers.js

```diff
@@ -0,0 +1,69 @@
+var HANDLERS = {
+	click: {
+	// Define all click handlers here
+		orderOpt: function(event) {
```

Comment:
If you set `Views >  Indentation > Indent using Spaces` in Sublime it will convert your tab characters to spaces so that it will look the same everywhere.